packages/amazonq/test/e2e/amazonq/featureDev.test.ts

```diff
@@ -18,6 +18,7 @@ describe('Amazon Q Feature Dev', function () {
 
     const prompt = 'Add blank.txt file with empty content'
     const codegenApproachPrompt = prompt + ' and add a readme that describes the changes'
+    const fileLevelAcceptPrompt = prompt + ' and add a license, and a contributing file'
```

Comment:
Let's use template literals here. Ex:

`${prompt} and add a license, and a contributing file`